components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-node/reporting-node.component.ts

```diff
@@ -95,7 +95,8 @@ export class ReportingNodeComponent implements OnInit, OnDestroy {
       this.store.select(controlDetailList)
     ]).pipe(takeUntil(this.isDestroyed))
     .subscribe(([detailsStatusSt, detailsListState]) => {
-      if (detailsStatusSt === EntityStatus.loadingSuccess && !isNil(detailsListState)) {
+      if (detailsStatusSt === EntityStatus.loadingSuccess && !isNil(detailsListState)
+      && this.controlList.control_elements !== undefined) {
```

Comment:
this will throw error when controlList is undefined, 
to avoid that, 
this.controlList && this.controlList.control_elements

Reply:
done